src/docfx/restore/RestoreWorkTree.cs

```diff
@@ -80,6 +80,9 @@ async Task AddWorkTrees()
                         await GitUtility.AddWorkTree(restorePath, rev, workTreePath);
                     }
 
+                    // update the last access time
+                    Directory.SetLastAccessTimeUtc(workTreePath, DateTime.UtcNow);
```

Comment:
why do we need to update the last access time manually?

Reply:
depends on different platform, it's supported differently, so we manually handle that:
https://superuser.com/questions/464290/why-is-cat-not-changing-the-access-time

Reply:
ok, thanks for the explanation